### PR TITLE
use modern optimizer in TF

### PR DESCRIPTION
**Pull request by jpata**
- switched optimizer to modern interface in TF
- remove multimodal loss from TF, I don't think it actually worked that well (and we don't have it in pytorch yet)
- update CMS simulation scripts
- update CMS truth definition (true electron becomes photon instead of neutral if no associated track)
- update CMS features: add cluster shape stuff
- fix warnings in pytorch data loading

### CODE REVIEW

**Comment by jpata** [comment]
New training (red) so far.
image
image
